test/integration/targets/azure_rm_containerregistry/tasks/main.yml

```diff
@@ -37,7 +37,19 @@
          Environment: Production
    register: output
 
- - name: Assert the ACS instance is well updated
+ - name: Create second container registry (to test facts)
```

Comment:
why create a registry delegated for test? why not use previously created one

Reply:
because there's only one created, and I want to test is a list is returned